nsv13/code/modules/munitions/ship_weapons/hybrid_weapons/hybrid_railgun.dm

```diff
@@ -138,7 +138,7 @@
 	var/obj/item/ship_weapon/ammunition/T = chambered
 	if(T)
 		var/final_velo = projectile_velo - ((100 - alignment) / 100) //Misalignment slows projectiles
-		linked.fire_projectile(T.projectile_type, target, FALSE, final_velo, null, TRUE)
+		linked.fire_projectile(T.projectile_type, target, speed=final_velo, user_override=TRUE, lateral=TRUE) //CHECK THIS CODE LATERAL WAS RECEIVING NULL, REPLACED TO TRUE
```

Comment:
Is there any reason why you set user_override to TRUE? I think user_override is supposed to be a mob to be saved in the projetile's firer var if that mob is different from the firing ship's gunner (as saved in the overmap structure's gunner var).

Reply:
Emphasis on *supposed*. A bit further down, in lines 622 to 625 it looks very much like it just overwrites it anyway so user_override is completely dead code as far as I can tell.